perfmetrics/scripts/testing_on_gke/examples/fio/parse_logs.py

```diff
@@ -145,42 +216,63 @@
 
       output[key]["records"][scenario][epoch - 1] = r
 
-  output_order = [
-      "read-64K",
-      "read-128K",
-      "read-1M",
-      "read-100M",
-      "read-200G",
-      "randread-1M",
-      "randread-100M",
-      "randread-200G",
+  scenario_order = [
+      "local-ssd",
+      "gcsfuse-generic",
+      "gcsfuse-no-file-cache",
+      "gcsfuse-file-cache",
   ]
-  scenario_order = ["local-ssd", "gcsfuse-no-file-cache", "gcsfuse-file-cache"]
 
   output_file = open("./output.csv", "a")
   output_file.write(
-      "File Size,Read Type,Scenario,Epoch,Duration (s),Throughput"
-      " (MB/s),IOPS,Throughput over Local SSD (%),GCSFuse Lowest Memory"
-      " (MB),GCSFuse Highest Memory (MB),GCSFuse Lowest CPU (core),GCSFuse"
-      " Highest CPU (core),Pod,Start,End\n"
+      "File Size,Read Type,Scenario,Epoch,Duration"
+      " (s),Throughput (MB/s),IOPS,Throughput over Local SSD (%),GCSFuse Lowest"
+      " Memory (MB),GCSFuse Highest Memory (MB),GCSFuse Lowest CPU"
+      " (core),GCSFuse Highest CPU"
+      " (core),Pod,Start,End,GcsfuseMoutOptions,BlockSize,FilesPerThread,NumThreads\n"
   )
 
-  for key in output_order:
-    if key not in output:
-      continue
+  for key in output:
     record_set = output[key]
 
     for scenario in scenario_order:
       for i in range(len(record_set["records"][scenario])):
-        r = record_set["records"][scenario][i]
-        r["throughput_over_local_ssd"] = round(
-            r["throughput_mb_per_second"]
-            / record_set["records"]["local-ssd"][i]["throughput_mb_per_second"]
-            * 100,
-            2,
-        )
-        output_file.write(
-            f"{record_set['mean_file_size']},{record_set['read_type']},{scenario},{r['epoch']},{r['duration']},{r['throughput_mb_per_second']},{r['IOPS']},{r['throughput_over_local_ssd']},{r['lowest_memory']},{r['highest_memory']},{r['lowest_cpu']},{r['highest_cpu']},{r['pod_name']},{r['start']},{r['end']}\n"
-        )
-
+        if ("local-ssd" in record_set["records"]) and (
+            len(record_set["records"]["local-ssd"])
+            == len(record_set["records"][scenario])
+        ):
+          try:
+            r = record_set["records"][scenario][i]
+            r["throughput_over_local_ssd"] = round(
+                r["throughput_mb_per_second"]
+                / record_set["records"]["local-ssd"][i][
+                    "throughput_mb_per_second"
+                ]
+                * 100,
+                2,
+            )
+          except:
+            print(
+                "failed to parse record-set for throughput_over_local_ssd."
+                f" record: {r}"
+            )
+            continue
```

Comment:
Similar to my other comment - can we refactor these statements into a different method?